dts/bindings/retention/retention-controller.yaml

```diff
@@ -0,0 +1,20 @@
+# Common fields for data retention drivers
```

Comment:
These properties need descriptions to understand what their purposes is.

Reply:
Added